holoviews/core/options.py

```diff
@@ -116,7 +116,7 @@ def __call__(self, *args, **kwargs):
     def clear(self, clone=False):
         return self.obj.opts(clone=clone)
 
-    def show(self):
+    def info(self):
```

Comment:
I'd like to expose whether to show defaults here and add a small docstring. I'd be happy to do so in another PR though.

Reply:
Agreed. Good idea for a separate PR.